corehq/apps/reports/filters/users.py

```diff
@@ -473,6 +475,28 @@ def get_default_selections(self):
         return [('t__6', _("[Web Users]"))]
 
 
+class PropertyFilter(BaseSingleOptionFilter):
+    label = ugettext_noop('Modified Property')
+    default_text = ugettext_noop('Select Property')
+    slug = 'property'
+
+    @property
+    def options(self):
+        if domain_has_privilege(self.domain, privileges.APP_USER_PROFILES):
+            user_data_label = _("Profile or User Data")
+        else:
+            user_data_label = _("User Data")
```

Comment:
Can we avoid this branching? and keep "Profile" separate?

Reply:
I don't love combining these either, but looking at `details['changes']` doesn't tell me whether it was the profile or other user data that changed, and I don't want the filter to be parsing the change message text, as discussed in https://github.com/dimagi/commcare-hq/pull/30064#issuecomment-881481494

Reply:
Actually, taking a step back.
My understanding was that "profile" was a part of the "user_data" itself. Is "profile" as special user data?
I think I am confused by the need to mention profile separately.

Reply:
You mean just remove the "Profile or" part? I think that projects that use profiles may be looking for a "profile" option.  But I can ask product about it (don't want to block this PR on that, though).

I'm guessing we'll iterate more on the right way to display profile and user data changes. "User Data" also isn't great: we don't use that text consistently in the UI, only in bulk upload.

Reply:
ya, my assumption is that profile is simply a `user_data`. The requirement that it needs to be treated as a special one, isn't great. If we can push back on it, that would be great.

> (don't want to block this PR on that, though).

Cool.

Reply:
> I'm guessing we'll iterate more on the right way to display profile and user data changes. "User Data" also isn't great: we don't use that text consistently in the UI, only in bulk upload.

Ya. Iteration can happen on how to show these on UI but I hope we won't need to store it differently.